README.md

```diff
@@ -33,6 +33,12 @@ mkdir ~/ws_nexus/src -p
 cd ~/ws_nexus/src/
 git clone git@github.com:osrf/nexus
 vcs import . < nexus/abb.repos
+
+# There are new features from Open-RMF that are required but not released into
+# Jazzy yet
+wget https://raw.githubusercontent.com/open-rmf/rmf/$RMF_BRANCH/rmf.repos
```

Comment:
Where is `$RMF_BRANCH` defined? Should this point to a fixed branch instead?

Reply:
https://github.com/osrf/nexus/pull/104/commits/2e51c9f82cfc53e3cfbe975b3b9a7e3a43eb64e2